test/membrane_live_web/channels/event_channel_test.exs

```diff
@@ -163,4 +145,18 @@ defmodule MembraneLiveWeb.EventChannelTest do
       )
     end
   end
+
+  defp create_user(google_claims) do
+    {:ok, user} = Accounts.create_user_if_not_exists(google_claims)
+    {:ok, token, _claims} = Tokens.custom_encode(user.uuid)
+    {:ok, user, token}
+  end
+
+  defp create_channel_connection(uuid, token) do
+    MembraneLiveWeb.EventSocket
+    |> socket("event_id", %{})
+    |> subscribe_and_join(MembraneLiveWeb.EventChannel, "event:#{uuid}", %{
+      token: token
+    })
+  end
```

Comment:
As other tests use this function you can move it to the `testUtills` module, so you won't duplicate code.